DFC.ServiceTaxonomy.Editor/Recipes/Service Taxonomy.recipe.json

```diff
@@ -2269,24 +2303,14 @@
                     "Name": "Skill",
                     "Settings": {},
                     "ContentPartFieldDefinitionRecords": [
-                        {
-                            "FieldName": "TextField",
-                            "Name": "ESCOSkillUri",
-                            "Settings": {
-                                "ContentPartFieldSettings": {
-                                    "DisplayName": "ESCO Skill Uri",
-                                    "Position": "0"
-                                }
-                            }
-                        },
                         {
                             "FieldName": "HtmlField",
                             "Name": "Description",
                             "Settings": {
                                 "ContentPartFieldSettings": {
                                     "DisplayName": "Description",
                                     "Editor": "Wysiwyg",
```

Comment:
Would you mind changing to the trumbowyg editor, so that we can standardize on it.